src/main/scala/com/redislabs/provider/redis/util/Logging.scala

```diff
@@ -0,0 +1,43 @@
+package com.redislabs.provider.redis.util
+
+import org.slf4j.{Logger, LoggerFactory}
+
+/**
+  * @author The Viet Nguyen
+  */
+trait Logging {
```

Comment:
Shouldn't we use the Spark native logger?

Reply:
Spark doesn't export its logging mechanism. This is more like copy-n-paste from `spark-core`. And I think it's a popular approaches among Spark satellite projects, like `spark-cassandra`, etc.

Reply:
I recall that their `Logging` was private/internal to spark, but it turns out that they made it public starting from some point. I have switched to spark's `Logging`.

Reply:
After more investigation it turns out that after Spark 2.0 they moved `Logging` to a private package. In the release notes they suggest to use slf4j directly. So I think it's better to use our own logger.